helm/templates/dist.yml

```diff
@@ -0,0 +1,156 @@
+{{ if .Values.dist }}
+{{ range $dist := .Values.dist.pops }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "dist.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "name" $ }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+    run: {{ include "dist.fullname" . }}
+spec:
+  type: ClusterIP
+  ports:
+  - port: 5353
+    targetPort: data
+    protocol: TCP
+    name: data
+  - port: 3300
+    targetPort: http-config
+    protocol: TCP
+    name: http-config
+  - port: 9090
+    targetPort: service
+    protocol: TCP
+    name: service
+  selector:
+    app: {{ include "dist.fullname" . }}
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "dist.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "name" $ }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+    app: {{ include "dist.fullname" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ include "dist.fullname" . }}
+  serviceName: "{{ include "dist.fullname" . }}"
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "name" $ }}
+        app.kubernetes.io/managed-by: {{ $.Release.Service }}
+        app.kubernetes.io/instance: {{ $.Release.Name }}
+        helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+        app: {{ include "dist.fullname" . }}
+    spec:
+      initContainers:
+      - name: init-sysctl
+        image: busybox
+        command:
+        - sysctl
+        - -w
+        - net.ipv6.conf.lo.disable_ipv6=0
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          privileged: true
+      containers:
+      - name: dist
+        image: {{ $.Values.dist.image.name }}:{{ $.Values.dist.image.tag }}
+        imagePullPolicy: {{ $.Values.dist.image.pullPolicy }}
+        env:
+        - name: CONTAINER_NAME
+          value: "dist"
+        - name: CONFIG_PORT
+          value: "3300"
+        - name: DIST_PORT
```

Comment:
This shouldn't be set here and we need to fix it if it still is

Reply:
Haven't been able to resolve this, I can set it to `""` and it works fine if that's preferable, otherwise I'll dig into it more